src/pages/SignUp.tsx

```diff
@@ -8,16 +8,22 @@ import useInputs from '../hooks/useInput';
 const SignUp = () => {
   const [userId, setuserId] = useInputs('');
   const [userPw, setuserPw] = useInputs('');
-  const [field, setField] = useState('FE');
+  const [part, setPart] = useState('');
```

Comment:
위에서 `userId`와 `userPw`로 네이밍 했으니 통일감을 위해 `userPart`는 어떨까요
이건 그냥 제안입니다

```suggestion
  const [userPart, setUserPart] = useState('');
```